.github/workflows/llvm.yml

```diff
@@ -0,0 +1,55 @@
+name: LLVM Test
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SPEC_SPLIT_DOTS: 160
+
+jobs:
+  llvm_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        llvm_version: ["13.0.0", "14.0.0", "15.0.6"]
+    name: "Test LLVM ${{ matrix.llvm_version }}"
+    steps:
+      - name: Checkout Crystal source
+        uses: actions/checkout@v3
+
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v3
+        with:
+          path: |
+            C:/Program Files/LLVM
```

Comment:
Since the workflow runs on Ubuntu, is this still relevant?

Reply:
I figure we could easily enable Windows as well if we want to. And then it's good to have this because the location is different in Windows. It doesn't hurt to keep it around and avoid missing out on this later.

Reply:
Makes sense, shouldn't it be written with backslashes though?

Reply:
I guess. I copied it right out of the instructions for the action. It doesn't really matter, though.